### DISCUSSION TITLE
Is there still support for PnP.Framework?

### DISCUSSION THREAD

**Discussion by web265p3** [pnp/pnpcore#1217] [General]
I see a lot of open discussions and replies in this repository, but the discussions in [PnP.Framework](https://github.com/pnp/pnpframework/discussions) repository seem to be almost dead.  

There are many questions, but no replies anymore. Also issues, but no replies neither.

We start to make more and more use of PnP.Core.Sdk and the PnP.Framework in our organization and are wondering if the PnP.Framework still has community support...
I also opened [a question](https://github.com/pnp/pnpframework/discussions/895) there, but I don't expect an answer, because even questions from the beginning of the year are still unanswered. 

Any infos?

When will PnP.Framework be shifted in PnP.Core.Sdk?

**Comment by jansenbe** [comment]
@web265p3 : we're gradually shifting more and more workloads to PnP Core SDK, main features still in PnP Framework are the provisioning engine and page modernization. None of these projects are "officially" supported, it happens that I spend my community time mainly with PnP Core SDK as I don't have the bandwidth to keep up with both. If there are features from Framework (outside of provisioning engine) that you need then let me know, if's easy to add I can get you unblocked.
